scripts/update_bootlint.js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+/* eslint-env shelljs */
+
+'use strict';
+
+require('shelljs/global');
+var https       = require('https');
+var fs          = require('fs');
+var path        = require('path');
+
+var version     = process.argv[2];
+
+var basedir     = path.join(__dirname, '..');
+var bootlintDir = path.join(basedir, 'public', 'bootlint', version);
+var UGLIFYJS    = path.join(basedir, 'node_modules/.bin/uglifyjs');
+
+if (!version) {
+    echo('Valid Bootlint version required.');
+    process.exit(1);
+}
+
+// strip leading 'v' if present
+version = version.replace(/^v/, '$1');
+
+if (test('-d', bootlintDir)) {
+    echo('Bootlint version already found.');
+    process.exit(1);
+}
+
+mkdir(bootlintDir);
+pushd(bootlintDir);
+
+
+https.get('https://raw.githubusercontent.com/twbs/bootlint/v' + version + '/dist/browser/bootlint.js', function (res) {
+    var statusCode = res.statusCode;
+    var error = { };
+    var file = fs.createWriteStream(path.join(bootlintDir, 'bootlint.js'));
+
+    if (statusCode !== 200) {
+        error = new Error('Request Failed.\nStatus Code: ' + statusCode);
+    }
+
+    if (error) {
```

Comment:
As an aside, I would simplify error checking by putting it all in a single conditional and remove the need for `error` all together...

```node
if (statusCode !== 200) {
   console.log(new Error('Request Failed.\nStatus Code: ' + statusCode).message);
   res.resume();
   return;
}
```

But totally up to you, if you prefer to keep them separate.